### PR TITLE
Added basic html

### PR DESCRIPTION
**Pull request by NataliiaSlate**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nataliia's CV</title>
+</head>
+
+<body style="background-color: #FAF0E6;">
+    <section>
+        <div>
```
Comment (Khousheish) [inline index.html]: right now this div serves no purpose. Its like a container inside a container. Section by itself is enough here. This applies to all places in the app.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            <h1>AQA Engineer and Full-stack Developer (in future)</h1>
+            <h2>Never say never</h2>
+            <img src="images/image.png" alt="Ups, the image is broken">
+        </div>
+    </section>
+    <section>
+        <div>
+            <h1>I'm Nataliia, and welcome to my space!</h1>
```
Comment (Khousheish) [inline index.html]: Usually its best practice to only have 1 h1 in the webpage as the main header. Its like a book that can only have 1 title. All titles under it are chapters for this book.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                    </tr>
+                </table>
+            </div>
+        </div>
+    </section>
+    <footer>
+        <div style="background-color: #fe5a75;">
+            <div>
+                <h3>
+                    <br>Embrace Challenges, Unlock Potential.<br><br>
```
Comment (Khousheish) [inline index.html]: Are the brs here to push the text a bit down? :D

**Review by Khousheish — APPROVED** [review]
Amazing work overall. All the comments are just nitpicks that are nice to know. I love how you used almost every single concept mentioned in the lecture and weren't scared of mixing stuff together(lists inside tables).

File: index.html (hunk 3 above)
Comment (NataliiaSlate) [reply index.html]: Yes, exactly

**Review by Khousheish — APPROVED** [review]

**Pull request merged**